openedx/core/djangoapps/user_api/management/commands/email_opt_in_list.py

```diff
@@ -46,20 +47,24 @@ def chunks(sequence, chunk_size):
 
 
 class Command(BaseCommand):
-    """Generate a list of email opt-in values for user enrollments. """
-
-    args = "<OUTPUT_FILENAME> <ORG_ALIASES> --courses=COURSE_ID_LIST --email-optin-chunk-size=CHUNK_SIZE"
+    """
+    Generate a list of email opt-in values for user enrollments.
+    """
     help = "Generate a list of email opt-in values for user enrollments."
-    option_list = BaseCommand.option_list + (
-        optparse.make_option('--courses ', action='store'),
-        optparse.make_option(
-            '--email-optin-chunk-size',
-            action='store',
-            type='int',
-            default=DEFAULT_CHUNK_SIZE,
-            dest='email_optin_chunk_size',
-            help='The number of courses to get opt-in information for in a single query.')
-    )
+
+    def add_arguments(self, parser):
+        parser.add_argument('file_path',
```

Comment:
Should we try to keep the original help text variable names via `metavar`?

Reply:
Done.